shopinvader_customer_multi_user/controllers/main.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Camptocamp SA (http://www.camptocamp.com).
+# @author Simone Orsi <simone.orsi@camptocamp.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.addons.shopinvader.controllers import main
+
+
+class InvaderController(main.InvaderController):
+    @classmethod
+    def _find_partner(cls, backend, partner_email):
+        binding = super()._find_partner(backend, partner_email)
```

Comment:
@simahawk What about traceability? All the actions into Odoo will be made in the name of the company and we forget the information of the real user behind the company. I don't see how we can preserve this traceability but IMO it's very important to keep a trace somewhere ....

Reply:
We can block company data editing if done via sub-users/contacts.
Or we can make this optional and track edits via tracking + notifications.

Reply:
IMO The most important is to keep also the information of the customer making of modifying the order.... 
The subject is quite complex. Can't we preserve the real customer as partner in the session, and only modify the necessary places where the information to be used would be those of the company. (invoice address, list of sale orders already done, delivery addresses, .....)

Reply:
would you be available today for call?